core/bin/external_node/src/init.rs

```diff
@@ -77,8 +77,8 @@ pub(crate) async fn ensure_storage_initialized(
         InitDecision::SnapshotRecovery => {
             anyhow::ensure!(
                 consider_snapshot_recovery,
-                "Snapshot recovery is required to proceed, but it is not enabled. Enable by supplying \
-                 `--enable-snapshots-recovery` command-line arg to the node binary, or reset the node storage \
+                "Snapshot recovery is required to proceed, but it is not enabled. Enable by setting \
```

Comment:
Could we move back to `--enable-snapshots-recovery` ?
I'm doing my best to reduce dependencies from env variables

Reply:
yes, I can revert that